python/pyspark/sql/connect/plan.py

```diff
@@ -32,6 +32,21 @@ class InputValidationError(Exception):
     pass
 
 
+def convert_value(v: Any) -> proto.Expression.Literal:
```

Comment:
we have this: https://github.com/apache/spark/blob/master/python/pyspark/sql/connect/column.py#L178

is it feasible to extract the common part and re-use?